conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/DialogueServiceGenerator.java

```diff
@@ -96,18 +100,21 @@ public Stream<JavaFile> generate(ConjureDefinition conjureDefinition) {
 
     private static Stream<JavaFile> generateFilesForService(
             boolean excludeDialogueAsyncInterfaces,
+            boolean generateDialogueEndpointErrorResultTypes,
             ServiceDefinition serviceDef,
             DialogueEndpointsGenerator endpointsGenerator,
             DialogueInterfaceGenerator interfaceGenerator,
             StaticFactoryMethodGenerator blockingGenerator,
             StaticFactoryMethodGenerator asyncGenerator) {
-        List<JavaFile> files = new ArrayList<>(/* initialCapacity= */ 3);
+        List<JavaFile> files = new ArrayList<>(/* initialCapacity= */ 4);
```

Comment:
why did this need to change?

Reply:
it doesn't need to change. this was a remnant of a previous iteration where I created a separate file for the result types.

will revert, thanks!